pkg/filter/stream/seata/filter.go

```diff
@@ -156,7 +156,7 @@ func (f *filter) Append(ctx context.Context, headers api.HeaderMap, buf buffer.I
 					}
 				}
 			} else {
-				// 服务宕机，没有 response，此时 header 为 request header
+				// service crash, there is no response，thus, the headers are request headers
```

Comment:
If service crashes, there will be no response. Thus the headers are request headers